include/boost/math/distributions/dirichlet.hpp

```diff
@@ -12,19 +12,20 @@
 
 // The Dirichlet distribution is a family of continuous multivariate probability
 // distributions parameterized by a vector 'alpha' of positive reals.
-// It is a multivariate generalization of the dirichlet distribution, hence its
-// alternative name of multivariate dirichlet distribution (MBD).
+// It is a multivariate generalization of the beta distribution, hence its
+// alternative name of multivariate beta distribution (MBD).
 // Dirichlet distributions are commonly used as prior distributions in
 // Bayesian statistics, and in fact the Dirichlet distribution is the
 // conjugate prior of the categorical distribution and multinomial distribution.
 
-#ifndef BOOST_MATH_DIST_DIRICHLET_HPP
-#define BOOST_MATH_DIST_DIRICHLET_HPP
+#ifndef BOOST_MATH_DISTRIBUTIONS_DIRICHLET_HPP
+#define BOOST_MATH_DISTRIBUTIONS_DIRICHLET_HPP
 
 #include <boost/math/distributions/fwd.hpp>
+#include <boost/multiprecision/float128.hpp>
```

Comment:
Don't include `boost/multiprecision/float128.hpp` in this file. Anyone who wants to use the `float128` type will #include it in their code. For instance, it should be in your unit tests, but not here, because it'll slow compilation.